manim/animation/creation.py

```diff
@@ -361,22 +381,10 @@ def __init__(
             run_time=run_time,
             lag_ratio=lag_ratio,
             rate_func=lambda t: -rate_func(t) + 1,
+            reverse=reverse,
             **kwargs,
         )
 
-    def begin(self) -> None:
-        if not self.reverse:
```

Comment:
I haven't tried this locally yet, but I don't see how the behavior of `Unwrite` is preserved if the logic is flipped and moved to `Write`. Doesn't `Unwrite` now do the same as `Write`?

(I'm on mobile, so it's likely that I've missed something.)

Reply:
That's true, and I initially did that. For some reason I thought that it somehow didn't work (I don't know why) and reverted my fix.